setup.py

```diff
@@ -14,7 +14,7 @@
       setup_requires=['setuptools_scm', 'setuptools_scm_git_archive'],
       zip_safe=False,
       packages=['Ska', 'Ska.Sun', 'Ska.Sun.tests'],
-      package_data={},
+      package_data={'Ska.Sun': ['data/*csv']},
```

Comment:
It's probably worth making this file smaller, esp. since it will eventually have the + and - columns.
```
In [10]: t = Table.read('Ska/Sun/data/pitch_rolldev.csv')
In [11]: t['pitch'] = t['pitch'].astype(np.float32)
In [12]: t['rolldev'] = t['rolldev'].astype(np.float32)
In [13]: t.write('junk.fits.gz', overwrite=True)
```
This compresses the file from 430k to 43k.

Reply:
I disagree on that one.

Reply:
This isn't really a file that is meant for human consumption. What is the advantage of CSV?  FITS is also faster to load.

Reply:
I just disagree with the "probably worth".  Granted a trivial cost to making the file smaller (and adding documentation someplace about how to make the file), but the price is losing transparency that includes github diffs (as in the FOT can see what we are using from github).

Reply:
OK. I don't think that the FOT or anyone will care about text diffs of a huge data file. That's not a useful way to see what has changed. If there is a diff then we would want a plot of some sort. If being complete with documentation, then you would start from referencing an email and document the step of translating that excel file to CSV. From there it is 3 lines of code to write out as FITS.

I just don't want to needlessly bloat this package to (eventually) nearly a Mb.

Reply:
OK.  I figured we'd just add a comment to the top of the CSV referencing the email and the original file name and it would be self-documenting, but fine if you want to save 400k compress it and stick the notes someplace else.  Where should those notes belong... just in the package readme?

Reply:
You can make a Python script `make_rolldev_file.py` at the top level in the repo. There you can put the actual code and documentation about provenance.

(In bigger repos we might hide that file in a `dev` or `utils` directory, but I think top-level is fine here).

Reply:
OK.  Though note that I'm not planning to set code to remake the csv we have now (which I just converted from xlsx or whatever using google sheets and then reduced to one column with astropy table).

Reply:
Sounds good.